src/autoTune.ml

```diff
@@ -230,6 +230,8 @@ let focusOnSpecification () =
   | ValidMemtrack
   | ValidMemcleanup -> (* Enable the memLeak analysis *)
     let memLeakAna = ["memLeak"] in
+    print_endline "Setting \"ana.malloc.unique_address_count\" to 1";
+    set_int "ana.malloc.unique_address_count" 1;
```

Comment:
Now that I think of it, this should probably only set this value to 1 if it is not set to a larger value before?

Reply:
Yes, it probably makes sense to change it only if it's < 1. I adapted the `set_int` call in https://github.com/goblint/analyzer/pull/1201/commits/b26010ac042143b4cdeb806b2a35b67054db1d76